actions/checkForAndPlaceOrder.ts

```diff
@@ -299,11 +290,35 @@ function _handleOrderBookError(
   return { shouldThrow: true };
 }
 
+type GetTradeableOrderWithSignatureResult =
+  | GetTradeableOrderWithSignatureSuccess
```

Comment:
Was there meant to be another element in the type that precedes this, or is this an extraneous 'or' operator?

Reply:
It's the linter. It is the same as not having it :) 
i think is prettier in action. I actually also found it weird, but don't argue to much with them.

You can see it in action here: https://codebeautify.org/typescript-formatter-online



<img width="1703" alt="image" src="https://github.com/cowprotocol/composable-cow/assets/2352112/f797affc-eff6-498c-a113-3d732bdd5ebe">